GOVERNANCE.md

```diff
@@ -1,137 +1,129 @@
-are modifications of governance, contribution policy, TSC membership,
-or release process.
+* Evangelism (this includes outward appearance, social media, etc)
+* Conferences
+* Meetups
+* Conduct
+* Moderation
 
-The intention of the agenda is not to approve or review all patches.
-That should happen continuously on GitHub and be handled by the larger
-group of Collaborators.
+* [Current list of CC members](./README.md#current-project-team-members)
 
-Any community member or contributor can ask that something be reviewed
-by the TSC by logging a GitHub issue. Any Collaborator, TSC member, or the
-meeting chair can bring the issue to the TSC's attention by applying the
-`tsc-review` label. If consensus-seeking among TSC members fails for a
-particular issue, it may be added to the TSC meeting agenda by adding the
-`tsc-agenda` label.
+All CC members are owners of the Ayo.js GitHub organization, and also have admin
+rights in the Ayo.js Discord server.
 
-Prior to each TSC meeting, the meeting chair will share the agenda with
-members of the TSC. TSC members can also add items to the agenda at the
-beginning of each meeting. The meeting chair and the TSC cannot veto or remove
-items.
+### Correspondence between TC and CC
 
-The TSC may invite additional persons to participate in a non-voting capacity.
+The TC and the CC are normally completely independent from each other, however
+in the event where an issue arises that crosses both of their areas of
+expertise, a joint action is required. In these events, none of the two groups
+may make a final decision until the other agrees.
 
-The meeting chair is responsible for ensuring that minutes are taken and that a
-pull request with the minutes is submitted after the meeting.
+## Elected Final Arbiters
 
-Due to the challenges of scheduling a global meeting with participants in
-several timezones, the TSC will seek to resolve as many agenda items as possible
-outside of meetings using
-[the TSC issue tracker](https://github.com/nodejs/TSC/issues). The process in
-the issue tracker is:
+Elected Final Arbiters (EFA) are Members of the Ayo.js projects that serve to
+resolve issues where there is no consensus. They are elected on a 1-year term,
+during which they can choose to resign at any time. In the event that this
+happens, reelections immediately taken into consideration.
 
-* A TSC member opens an issue explaining the proposal/issue and @-mentions
-  @nodejs/tsc.
-* After 72 hours, if there are two or more `LGTM`s from other TSC members and no
-  explicit opposition from other TSC members, then the proposal is approved.
-* If there are any TSC members objecting, then a conversation ensues until
-  either the proposal is dropped or the objecting members are persuaded. If
-  there is an extended impasse, a motion for a vote may be made.
+There should always be multiple EFAs, a recommended number is 3-5.
+
+EFAs have the intellectual authority to overrule the TC and the CC on issues
```

Comment:
As per discord discussion, I think "decisive vote" is a better term here; It implies authority only in case of lack of consensus, which seems to be a core of EFA.